faiss/impl/ScalarQuantizer.cpp

```diff
@@ -256,6 +293,19 @@ struct Codec6bit {
 
 #endif
 
+#ifdef __AVX512F__
+    static FAISS_ALWAYS_INLINE __m512
+    decode_16_components(const uint8_t* code, int i) {
+        __m256 v0 = decode_8_components(code, i);
```

Comment:
Please implement 16 components instead of 2x8.
Alternatively, please refer to https://github.com/zilliztech/knowhere/blob/main/thirdparty/faiss/faiss/impl/ScalarQuantizerCodec_avx512.h , which has it all implemented

Reply:
Fixed.